app/views/users/show.html.erb

```diff
@@ -76,8 +85,8 @@
             <span class="profile-header__meta__item -ml-1">
               <% Profile.special_social_link_attributes.each do |attribute| %>
                 <% if @user.send(attribute).present? %>
```

Comment:
interesting that Rubocop didn't catch this one, we're using `.send` here and `.public_send` on the following line

Reply:
This isn't really related to this PR, but I can change it if you want?